src/databricks/labs/remorph/reconcile/execute.py

```diff
@@ -87,12 +88,12 @@ def main(*argv) -> None:
     try:
         recon_output = recon(
             ws=w,
-            spark=DatabricksSession.builder.getOrCreate(),
+            spark=DatabricksSession.builder.sdkConfig(w.config).getOrCreate(),
```

Comment:
check this.

Reply:
I observed an issue here: If you don't pass the workspace `SDK config` to the `Databricks Session builder`, it uses the default profile and spins us a new cluster.